pkg/kedge/grpc/integration_test.go

```diff
@@ -238,12 +259,18 @@ func (s *BackendPoolIntegrationTestSuite) SetupSuite() {
 	// Make ourselves the resolver for SRV for our backends. See Lookup function.
 	s.originalSrvResolver = srvresolver.ParentSrvResolver
 	srvresolver.ParentSrvResolver = s
+
+	// Make ourselves the A resolver for backends for the Addresser.
+	s.originalAResolver = common.DefaultALookup
```

Comment:
you can do `s.originalAResolver, common.DefaultALookup = common.DefaultALookup, lookupAddr` but not a blocker